src/components/esn-popover/esn-popover.tsx

```diff
@@ -0,0 +1,79 @@
+import { Component, ComponentInterface, Host, Element, State, h, Method } from '@stencil/core';
+import { createPopper } from '@popperjs/core/lib/popper-lite';
+import flip from '@popperjs/core/lib/modifiers/flip';
+import preventOverflow from '@popperjs/core/lib/modifiers/preventOverflow';
+
+@Component({
+  tag: 'esn-popover',
+  styleUrl: 'esn-popover.scss',
+  shadow: true
+})
```

Comment:
- is this missing `;` ?
- add a blank line after.

Reply:
No it's not missing `;` or a blank line. It's just a [TypeScript decorator](https://www.typescriptlang.org/docs/handbook/decorators.html#class-decorators), and it's supposed to be written this way.

Reply:
ok